pkg/app/app.go

```diff
@@ -498,7 +498,7 @@ func (c *App) fetchUserCredentials(user accounts.User) (*oauth2.Token, error) {
 	}
 	if !tk.Valid() {
 		// Refresh the token and store it in the db.
-		tks := c.oauth2Helper.TokenSource(context.TODO(), tk)
+		tks := c.oauth2Helper.OAuthConfig.TokenSource(context.TODO(), tk)
```

Comment:
Ditto, I think we should care when `helper.OAuthConfig` is nil. Unless we have a check on OAuthConfig, we would face at unexpected behavior.